### DISCUSSION TITLE
登録・ログインページの説明について

### DISCUSSION THREAD

**Discussion by daiki-nishimura** [flutter-osaka-dev/flutter-chat#7] [Q&A] [answered]
登録・ログインページの説明にて以下のように紹介がありますが、register_pageではなくlogin_page.dartではないでしょうか？

// TODO: ログインページが実装できたらコメントを外す下のコメントを外すと、そのボタンを押した時にログインページに飛ぶようになります。その際、"register_page.dart"

**Comment by dshukertjr** [comment, marked as answer]
おっしゃる通りでした！修正しておきますね！見つけていただきありがとうございます！
